test-network-function/certification/suite.go

```diff
@@ -117,7 +117,7 @@ func testContainerCertificationStatus() {
 				}
 				allContainersToQueryEmpty = false
 				ginkgo.By(fmt.Sprintf("Container %s/%s should eventually be verified as certified", c.Repository, c.Name))
-				isCertified := waitForCertificationRequestToSuccess(getContainerCertificationRequestFunction(c.Repository, c.Name), apiRequestTimeout)
+				isCertified := waitForCertificationRequestToSuccess(getContainerCertificationRequestFunction(c), apiRequestTimeout)
```

Comment:
In addition to basic certification result, maybe we could return the grade of the image and fail with recommendation to upgrade or pass with warning, if the image is too old. For instance, if the grade is >C we could fail, since older images a subject to security issues.
https://catalog.redhat.com/api/containers/v1/repositories/registry/registry.access.redhat.com/repository/rhel8/nginx-116/images?filter=freshness_grades.grade==D

Reply:
#596 raised